requirements/base.txt

```diff
@@ -12,7 +12,7 @@ git+https://github.com/edx/edx-submissions.git@1.1.4#egg=edx-submissions==1.1.4
 boto>=2.32.1,<3.0.0
 celery==3.1.18
 defusedxml==0.4.1
-django>=1.8.17,<1.9
+django<1.9a0 # 'a' resolves known bug on gemnasium
```

Comment:
@robrap It's a bug in Gemnasium - I couldn't find any documentation and was apprised of it by a Gemnasium support person. I wrote a (very brief) summary on the JIRA ticket. I guess I meant "known bug" as in "known by them".